src/NClient.Tools/NClient.DotNetTool/CommandLineOptions.cs

```diff
@@ -0,0 +1,28 @@
+using CommandLine;
+
+namespace NClient.DotNetTool
+{
+    // ReSharper disable once ClassNeverInstantiated.Global
+    public class CommandLineOptions
+    {
+        public CommandLineOptions(string spec, string outputPath, string @namespace, string facadeName)
+        {
+            Spec = spec;
+            OutputPath = outputPath;
+            Namespace = @namespace;
+            FacadeName = facadeName;
+        }
+
+        [Option(shortName: 's', longName: "spec", Required = true, HelpText = "The OpenAPI spec file to use. Paths are relative to the project directory.")]
```

Comment:
Getting remote specifications is a awesome feature, you need to tell about it in the description

Reply:
Done